Makefile

```diff
@@ -66,6 +66,42 @@ lint:  ## Run lint on the package
 		cd "internal/build/" && go mod tidy && go mod download && go vet ./...; \
 	}
 
+package := "prettier"
```

Comment:
Is there really no linter written in go so that we don't need to require `npm`?

Reply:
There probably exists such linter, but I came from another issue #258, since there was no questions of the tool used, I decided to use it to standardize. If you need the instrument to be on go, of course I can take the trouble to tune it ...

Reply:
I searched up everywhere in the web, there is no such package written in go, that can lint markdown files. Even if such exists, it is no longer maintained. 

Forced to admit ChatGPT to admit that there is no such linter after a long conversation...

```
I apologize for my previous responses. You are correct that there is currently no package written in Go that can lint Markdown files.

While there are some Markdown linters that integrate with Go, such as mdl, they are not written in Go themselves.

Again, I apologize for any confusion I may have caused and I hope this clears up any confusion.
```

Reply:
ok, use the one we have

please see other comments/asks

Reply:
Summary of our conversation:

* Fetch source code with fetch-depth - 1
* Changed CI linter option to check only and fail if files are invalid
* Add script to Makefile to prettify markdown files

Makefile script checks format of markdown files, in case of invalid files, asks user to decide auto format or leave as is